server/data/apps.json

```diff
@@ -2111,5 +2111,31 @@
         "elementURL": "http://focus.hk.cn"
       }
     ]
+  },
+  {
+    "appName": "iCatcher! Podcast Player",
+    "appType": [
+      "app",
+      "podcast player"
+    ],
+    "appUrl": "https://apps.apple.com/app/apple-store/id414419105",
+    "appIconUrl": "",
```

Comment:
Hey @joeisanerd , It looks like you left `appIconUrl` blank. Do you have a square png icon for your app that you can add to `server/assets/` and then reference here?

Reply:
I'll fix it.

Reply:
Thanks.